markdown/cli.ts

```diff
@@ -161,15 +165,17 @@ program
         string,
         { offset: number; invalid: []; unhandled: [] }
       >();
-      // replace '\' with '/' for Windows path
-      const slugPrefix = args.folder.toLowerCase().replace(/\\/g, "/");
+      const folderPrefix = folder?.slice(0, -1);
```

Comment:
I didn't yet find the time to test this. But from reading it, it seems like this is removing the trailing slash again?

Reply:
Because the `Document.find` will return docs which was in `**path/to/folder/**`: <https://github.com/mdn/yari/pull/6608/files#diff-8b04a3903730a363dd6402fa699fa3ecb6b241c1be608b573b194fe2440cbf54R146-R153>

And the `doc.fileInfo.folder` will end with a slash (like `web/css/colon-moz-only-whitespace`), to match with folder, we can only match prefix here (only match `path/to/folder/**`).

Reply:
or we can add a slash in the end of `doc.fileInfo.folder`

Reply:
maybe add a trailing slash is better, fixed it.